content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Set up security alerts"
+weight: 500
+toc: true
+nd-content-type: how-to
+nd-product: NGINX One
+---
+
+With this page, you'll learn how to set up alerts in F5 Distributed Cloud. Once configured, you'll see the CVEs and insecure configurations associated with your NGINX fleet. These instructions are intended for those responsible for keeping their NGINX infrastructure and application traffic secure. It assumes you know how to:
```

Comment:
Should we spell out CVE on first mention? Or is this a well-known domain acronym?

Reply:
Maybe I have the "curse of knowledge" on this one. It's not spelled out in the UI. You're welcome to say "that's not good enough". 

<img width="534" height="437" alt="Screenshot 2025-07-16 at 7 14 36 AM" src="https://github.com/user-attachments/assets/588b1394-c37d-48a6-bdee-b17e3311671e" />